tests/third_party_tests.rs

```diff
@@ -0,0 +1,85 @@
+use std::env;
+use std::fs;
+use std::path::PathBuf;
+use std::process::Command;
+
+use php_parser_rs::Lexer;
+use php_parser_rs::Parser;
+
+#[test]
+fn third_party_php_standard_library() {
+    test_repository(
+        "php-standard-library",
+        "https://github.com/azjezz/psl.git",
+        "2.2.x",
+        &["src", "tests", "examples"],
+    );
+}
```

Comment:
once tests pass for this, we can add more repositories.

Reply:
also a note: all third-party tests must start with `third_party` to ensure that they get filtered out